doc/develop/fixing_data.rst

```diff
@@ -353,3 +353,19 @@ For example for monthly data, place the files in the ``/Tier3/MSWEP/latestversio
   For monthly data (V220), the data must be postfixed with the date, i.e. rename ``global_monthly_050deg.nc`` to ``global_monthly_050deg_197901-201710.nc``
 
 For more info: http://www.gloh2o.org/
+
+.. _extra-facets-fixes:
+
+Use of extra facets in fixes
+============================
+In fixes, extra facets can be used to mold data into the form required by the
```

Comment:
Could you add a link here to the extra facets section in the configuration chapter?